pkg/cases/handler.go

```diff
@@ -8,6 +8,9 @@ import (
 type CaseHandler struct {
 }
 
+type MedicineHandler struct {
```

Comment:
Ahh, no need for this, merge it in `CaseHandler` will be ok

Reply:
but if merge into `CaseHandler`, how to group `GetMedicines`'s subgroup in `router.go`?

Reply:
Well, it's ok if you split them. Just keep it the way is it.